frontend/src/components/sellListing/NewListingTopBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
```

Comment:
There is already a component for this. Look for the <PageHeader> and make use of it, instead of re-implementing it.

Reply:
@kthristow  The component is called <PageHeader />. Please re-use it. It has exposed header and children props.